sql/core/src/test/scala/org/apache/spark/sql/execution/joins/ExistenceJoinSuite.scala

```diff
@@ -103,7 +103,7 @@ class ExistenceJoinSuite extends SparkPlanTest with SharedSparkSession {
       ProjectExec(output, FilterExec(condition, join))
     }
 
-    test(s"$testName using ShuffledHashJoin") {
+    testWithWholeStageCodegenOnAndOff(s"$testName using ShuffledHashJoin") { _ =>
```

Comment:
This test enforcement looks orthogonal to this PR (`Avoiding unncessary code generation`). Can we do this as a test PR independently? We can merge it separately to the other branches.

Reply:
@dongjoon-hyun - if the purpose of separating PRs is to merge change to other branches as well, I can submit another PR for all test-related change, i.e. the change in `InnerJoinSuite.scala`, `OuterJoinSuite.scala` and `ExistenceJoinSuite.scala`.

Do we want to do that? These test changes are valid to be merged into `3.1` branch as well.

Reply:
Yes, I want to merge the test improvement into branch-3.1.

Reply:
SPARK-34702 is for Apache Spark 3.2.0.

Reply:
@dongjoon-hyun - sure, updated.